README.md

```diff
@@ -195,6 +195,43 @@ Done with this data source? Remove it with the following command:
 DROP DATABASE my_pg;
 ```
 
+## Supported data sources
+
+| Source                 | Read | Write | Table Function | External Table | External Database | Supported Object Stores             |
+| ---------------------- | :--: | :---: | :------------: | :------------: | ----------------- | ----------------------------------- |
+| **Databases**          |  --  |  --   |       --       |       --       | --                | --                                  |
+| MySQL                  |  ✅  |  ✅   |       ✅       |       ✅       | ✅                | ➖                                  |
+| PostgreSQL             |  ✅  |  ✅   |       ✅       |       ✅       | ✅                | ➖                                  |
+| Microsoft SQL Server   |  ✅  |  🚧   |       ✅       |       ✅       | ✅                | ➖                                  |
+| MongoDB                |  ✅  |  🚧   |       ✅       |       ✅       | ✅                | ➖                                  |
+| Snowflake              |  ✅  |  ✅   |       ✅       |       ✅       | ✅                | ➖                                  |
+| BigQuery               |  ✅  |  ✅   |       ✅       |       ✅       | ✅                | ➖                                  |
+| Cassandra/ScyllaDB     |  ✅  |  🚧   |       ✅       |       ✅       | ✅                | ➖                                  |
+| ClickHouse             |  ✅  |  ✅   |       ✅       |       ✅       | ✅                | ➖                                  |
+| MariaDB _(via mysql)_  |  ✅  |  ✅   |       ✅       |       ✅       | ✅                | ➖                                  |
+| DuckDB                 |  🚧  |  🚧   |       🚧       |       🚧       | 🚧                | ➖                                  |
+| Oracle                 |  🚧  |  🚧   |       🚧       |       🚧       | 🚧                | ➖                                  |
+| SQLite                 |  🚧  |  🚧   |       🚧       |       🚧       | 🚧                | ➖                                  |
+| ADBC                   |  🚧  |  🚧   |       🚧       |       🚧       | 🚧                | ➖                                  |
+| ODBC                   |  🚧  |  🚧   |       🚧       |       🚧       | 🚧                | ➖                                  |
+| **File Formats**       |  --  |  --   |       --       |       --       | --                | --                                  |
```

Comment:
@vrongmeal Can you double check which ones we support writes for?

Reply:
Sure! I’ll take a look.